app/views/publish/courses/accredited_body/new.html.erb

```diff
@@ -4,6 +4,8 @@
   <%= govuk_back_link_to(@back_link_path) %>
 <% end %>
 
+<%= render CaptionText::View.new(text: "Add course") %>
```

Comment:
Could we put the "Add course" text into `I18n` so we can update it once for different areas

Reply:
This is also done [here](https://github.com/DFE-Digital/publish-teacher-training/pull/2998/commits/c18acde1c223e5d4eeaae0ea0218ece5e45eb0a8)